worker/test/src/RTC/Codecs/TestVP8.cpp

```diff
@@ -292,4 +292,32 @@ SCENARIO("process VP8 payload descriptor", "[codecs][vp8]")
 		REQUIRE(forwarded->pictureId == 1);
 		REQUIRE(forwarded->tl0PictureIndex == 1);
 	}
+
+	SECTION("drop packets that belong to other temporal layers after rolling over pictureID")
+	{
+		RTC::Codecs::EncodingContext::Params params;
+		params.spatialLayers  = 0;
+		params.temporalLayers = 2;
+		Codecs::VP8::EncodingContext context(params);
+		context.SyncRequired();
+
+		context.SetCurrentTemporalLayer(0);
+		context.SetTargetTemporalLayer(0);
+
+		// Frame 1
```

Comment:
```suggestion
		// Frame 1.
```

Reply:
Should I update the same comments in TestVP9?

Reply:
Yes please, dot at the end in all comments. ☺️